src/globals.ts

```diff
@@ -0,0 +1,27 @@
+import chokidar from 'chokidar';
+import chalk from 'chalk';
+
+const GreenLog = chalk.hex('#00FF00');
+const OrangeLog = chalk.hex('#FFA500');
+
+export const specWatcher = (filePath: string, handler: any, handlerName: string) => {
+  const CHOKIDAR_CONFIG = {
+    awaitWriteFinish: true
+  };
+  const WATCH_MESSAGES = {
+    logOnStart: (filePath: string) => console.log(GreenLog(`Watching AsyncAPI file at ${filePath}`)),
+    logOnChange: (handlerName: string) => console.log(OrangeLog(`Change detected, running ${handlerName}`)),
+  };
```

Comment:
Please move that arrays outside `specWatcher` function because you don't need to recreate them on each evaluation of `specWatcher` function :)